src/vos/tests/vts_pm.c

```diff
@@ -1220,6 +1220,7 @@ cond_updaten_op_(void **state, daos_handle_t coh, daos_unit_oid_t oid,
 	vts_dtx_begin(&oid, coh, epoch, 0, &dth);
 	rc = vos_obj_update_ex(coh, oid, epoch, 0, flags, &dkey, n, iod, NULL,
 			       sgl, dth);
+	D_DEBUG(DB_TRACE, "obj update %d \n", rc);
```

Comment:
(style)  unnecessary whitespace before a quoted newline

Reply:
probably fix this one